### ISSUE TITLE
Stabilize webpack chunk output so names are predictable and consistent between releases

### ISSUE THREAD

**Issue by senadir** [woocommerce/woocommerce-gutenberg-products-block#8454]
Cart and Checkout blocks emit a bunch of chunks that are dynamically loaded on demand if needed. However, the webpack config for that is a bit random and the files names are unpredictable, this is because webpack aggressively tries to match similar functions, this results in large file names or hashed files.

A fix for this is to tweak the naming, grouping, and threshold for such chunks so it's more predictable and stable.

This is not a bug in itself but causes other bugs in systems that rely on static file names like cache and translations.

We attempted to fix this before here https://github.com/woocommerce/woocommerce-blocks/pull/6420 but reverted it here https://github.com/woocommerce/woocommerce-blocks/pull/6456 to not break translation.

I think we can figure out a solution that achieves both (possibly a temporary hack).

**Comment by senadir** [comment]
This is more significant in webpack 5 and will cause more problems, output are now more random and we have more files:

```
1395-frontend.js
1478.js
1495-frontend.js
1534-frontend.js
1713-style.js.LICENSE.txt
1714-frontend.js
1926-frontend.js
1952.js
3611.js
3775.js
3810-frontend.js
4846-frontend.js
4928-frontend.js
5143-rtl.css
5143.css
5650-frontend.js
5882-frontend.js
5918.js
6218-rtl.css
6218.css
6252-frontend.js
6253-frontend.js
6928-frontend.js
7105-frontend.js
7333-frontend.js
7821-rtl.css
7821.css
7846-style.js.LICENSE.txt
7856-frontend.js
799-frontend.js
8730-frontend.js
882-rtl.css
882.css
9406-rtl.css
9406.css
```

**Comment by senadir** [comment]
Closed by work done in the #8013
